test/k8s-integration/cluster.go

```diff
@@ -182,11 +183,25 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string, use
 	}
 
 	cmd = exec.Command("gcloud", cmdParams...)
-	err = runCommand("Staring E2E Cluster on GKE", cmd)
+	err = runCommand("Starting E2E Cluster on GKE", cmd)
 	if err != nil {
 		return fmt.Errorf("failed to bring up kubernetes e2e cluster on gke: %v", err)
 	}
 
+	// Because gcloud cannot disable addons on cluster create, the deployment has
+	// to be disabled on update.
+	clusterVersion := mustGetKubeClusterVersion()
```

Comment:
This update should be performed if we detect a pdcsi-node daemonset in kube-system namespace.(as this indirectly indicates that the component is enabled)